src/Components/BureauPage/PositionManager/PositionManager.jsx

```diff
@@ -146,24 +147,26 @@ const PositionManager = props => {
     q: textInput || textSearch,
   };
 
-  const noBureausSelected = selectedBureaus.filter(f => f).length < 1;
-  const noOrgsSelected = selectedOrgs.filter(f => f).length < 1;
+  // const noBureausSelected = selectedBureaus.filter(f => f).length < 1;
+  const noBureausSelected = selectedBureaus.length === 0;
```

Comment:
```suggestion
  const noBureausSelected = selectedBureaus?.length === 0;
```

might be unnecessary - but, its always better to be safe